app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml

```diff
@@ -0,0 +1,13 @@
+- if user_signed_in? && @person != current_user.person
+  %select{:id => "user_aspects", 'data-person-id' => @person.id}
+    %option{:value => 'listcover', :class => 'listcover', :disabled => 'true', :selected => 'true'}
+      = t("add_contact")
+    - contact = current_user.contact_for(@person)
+    - current_user.aspects.each do |aspect|
+      - if !contact.nil? && contact.in_aspect?(aspect)
+        - membership_id = contact.aspect_memberships.where(:aspect_id => aspect.id).first.id
```

Comment:
`.limit(1).pluck(:id)` instead of `.first.id`, avoids instantiating the record just for a single attribute.